shared/constants/multichain/networks.ts

```diff
@@ -34,31 +39,48 @@ export type MultichainProviderConfig = ProviderConfigWithImageUrl & {
 export type MultichainNetworkIds = `${MultichainNetworks}`;
 
 export enum MultichainNetworks {
-  BITCOIN = 'bip122:000000000019d6689c085ae165831e93',
-  BITCOIN_TESTNET = 'bip122:000000000933ea01ad0ee984209779ba',
+  BITCOIN = BtcScope.Mainnet,
+  BITCOIN_TESTNET = BtcScope.Testnet,
+  BITCOIN_SIGNET = BtcScope.Signet,
 
-  SOLANA = 'solana:5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp',
-  SOLANA_DEVNET = 'solana:EtWTRABZaYq6iMfeYKouRu166VU2xqa1',
-  SOLANA_TESTNET = 'solana:4uhcVJyU9pJkvQyS88uRDiswHXSCkY3z',
+  SOLANA = SolScope.Mainnet,
+  SOLANA_DEVNET = SolScope.Devnet,
+  SOLANA_TESTNET = SolScope.Testnet,
 }
 
+// TODO: This data should be provided by the snap
+export const MULTICHAIN_NETWORK_TO_ACCOUNT_TYPE_NAME: Record<
+  CaipChainId,
+  string
+> = {
+  [BtcScope.Mainnet]: 'Bitcoin',
+  [BtcScope.Testnet]: 'Bitcoin testnet',
+  [SolScope.Mainnet]: 'Solana',
+  [SolScope.Testnet]: 'Solana',
+  [SolScope.Devnet]: 'Solana',
+} as const;
+
 export const MULTICHAIN_ACCOUNT_TYPE_TO_MAINNET = {
   [BtcAccountType.P2wpkh]: MultichainNetworks.BITCOIN,
   [SolAccountType.DataAccount]: MultichainNetworks.SOLANA,
 } as const;
 
 export const MULTICHAIN_NETWORK_TO_NICKNAME: Record<CaipChainId, string> = {
   [MultichainNetworks.BITCOIN]: 'Bitcoin',
-  [MultichainNetworks.BITCOIN_TESTNET]: 'Bitcoin (testnet)',
+  [MultichainNetworks.BITCOIN_TESTNET]: 'Bitcoin Testnet',
+  [MultichainNetworks.BITCOIN_SIGNET]: 'Bitcoin Signet',
   [MultichainNetworks.SOLANA]: 'Solana',
-  [MultichainNetworks.SOLANA_DEVNET]: 'Solana (devnet)',
-  [MultichainNetworks.SOLANA_TESTNET]: 'Solana (testnet)',
+  [MultichainNetworks.SOLANA_DEVNET]: 'Solana Devnet',
+  [MultichainNetworks.SOLANA_TESTNET]: 'Solana Testnet',
 };
```

Comment:
Nit: We could also make this `as const` (since we've added it to some other maps)

Reply:
Added the const, but I'd rather not change UI facing parts

Reply:
https://github.com/MetaMask/metamask-extension/pull/31702/commits/951c0efd729e180d881e0c51ffdc75f9f75a2960